website/src/pages/product-ecosystem.tsx

```diff
@@ -29,7 +106,188 @@ function Index() {
             </div>
           </div>
         </div>
-        <div className={common.container}>
+
+        {/* Background image Credit: https://www.toptal.com/designers/subtlepatterns/whirlpool/ */}
+        <div
+          className={common.section}
+          style={{
+            paddingTop: "48px",
+            paddingBottom: "60px",
+            backgroundImage: "url('/img/whirlpool.webp')",
+          }}
+        >
+          <div className={`${common.container} ${styles.roadmapSection}`}>
+            <div className={common.title}>Roadmap</div>
+            <div className={styles.roadmapContainer}>
+              {/* Products on Radar */}
+              <div className={styles.roadmapColumn}>
+                <div className={styles.roadmapColumnTitle}>
+                  <LocateFixed className={styles.titleIcon} /> Products on our
+                  Radar <div className={styles.offCenterBalance}></div>
+                </div>
+                <div
+                  className={`${styles.roadmapItemContainer} ${styles.productsOnRadar}`}
+                >
+                  <Scrollbars
+                    autoHeight
+                    autoHeightMax={500}
+                    className={styles.scrollbar}
+                  >
+                    {RoadmapItems.products_on_radar.map((item, index) => {
+                      return (
+                        <button
+                          key={index}
+                          className={styles.itemBtn}
+                          data-tip={formatTooltipContent(item)}
+                          data-for="rightTip"
+                          onClick={() => {
+                            setModalVisible(true)
+                            setModalContent(item)
+                          }}
+                        >
+                          {item[0]}
+                        </button>
+                      )
+                    })}
+                  </Scrollbars>
+                </div>
+              </div>
```

Comment:
The columns seem pretty identical? Maybe extract into a component?

Reply:
Great idea! 👏✨